lib/onfleet/tasks.js

```diff
@@ -1,20 +1,44 @@
+const moment = require('moment');
+
 class Tasks {
 
     constructor(client) {
         this.client = client;
     }
 
     /**
-     * Gets a single task for the given id or all tasks that were completed in the last 24 hours, scheduled in the
-     * next 24 hours, or created but not completed.
+     * Gets a single task for the given id
      *
-     * http://docs.onfleet.com/v2.0/docs/tasks#get-single-task
+     * http://docs.onfleet.com/docs/tasks#get-single-task
      *
      * @param {string} id
      */
-    get(id = null) {
-        let path = id ? `/tasks/${id}` : `/tasks`;
-        return this.client.get(path);
+    get(id) {
+        console.log('in get');
+        if(!id) {
+            throw new Error('No id given');
+        }
+        return this.client.get(`/tasks/${id}`);
+    }
+
+    /**
+     * Gets all tasks. Result is paginated.
+     * Result is of form - {lastId: 'asdad', tasks: [...]} where lastId is the id of the last element in the page.
+     * If null, the last page has been reached.
+     * @returns {Promise}
+     */
+    all(from, to = null, lastId = null) {
+        let qs = {from: from.valueOf()};
```

Comment:
Outside of the context of `onDay`, I'd expect `all()` to work without a `from` unless the Onfleet API requires it?
The `valueOf()` would cause an error.